web/beacon-app/src/features/projects/components/ChangeOwner/ChangeOwnerForm.tsx

```diff
@@ -31,6 +32,17 @@ const ChangeOwnerForm = ({ handleSubmit, initialValues }: ChangeOwnerFormProps)
     }));
   };
 
+  const optionsAvailable = formatMembers()?.filter(
+    (opt: any) =>
+      opt?.value !== values.current_owner.value && opt?.status !== MemberStatusEnum.PENDING
+  );
+
+  useEffect(() => {
+    if (optionsAvailable?.length === 0) {
+      toast.error(t`There are no other members to select as the new owner.`);
```

Comment:
I noticed that this error appears twice. Is it possible to make sure it only appears once after a user opens the modal?